tests/antithesis/test-template/robustness/traffic/main.go

```diff
@@ -159,10 +158,42 @@ func simulateTraffic(ctx context.Context, hosts []string, ids identity.Provider,
 		reports = append(reports, c.Report())
 		mux.Unlock()
 	}(compactClient)
+	dd := time.Duration(robustnessrand.RandRange(3, 7)) * traffic.DefaultCompactionPeriod
```

Comment:
There should be more variability and we should introduce some jitter or all defrags will be executed at the same time.

Reply:
Yep, jitter added.